relay-event-schema/src/processor/attrs.rs

```diff
@@ -123,14 +123,17 @@ impl MaxChars {
             MaxChars::Symbol => 256,
             MaxChars::Path => 256,
             MaxChars::ShortPath => 128,
-            // these are from constants.py or limits imposed by the database
-            MaxChars::Logger => 64,
-            MaxChars::Email => 75,
-            MaxChars::Culprit => 200,
-            MaxChars::TagKey => 32,
-            MaxChars::TagValue => 200,
-            MaxChars::Environment => 64,
-            MaxChars::Distribution => 64,
+            // These limits below were initially derived from Sentry's original implementation in
+            // Python. Before changing any of these limits, check if the values are inserted into
```

Comment:
> These limits below were initially derived from Sentry's original implementation in Python.

With the increase in tag keys, this no longer holds. I'd remove this sentence and leave the rest, but no strong opinions.

Reply:
I can try to clarify the wording here, but the sentence should hold. These limits _were originally_ derived from constants, and they are evolving. The intent here is to clarify on:
- The origins of the limit are outside of Relay's codebase and history
- There are technical reasons for some of the limits
- Any change needs to be carefully reviewed.